cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -1,3 +1,6 @@
+#ifdef ncdf
```

Comment:
Does this mean we still need ncdf passed in the compile option (-Dncdf), then use it to define USE_NETCDF, I am wondering if we can directly use USE_NETCDF as compile option (-DUSE_NETCDF) since all the rest of code only uses "USE_BETCDF" MACRO.

Reply:
The current comp_ice.backend.libcice and backend.clean in the branch I am preparing for the CICE6 commit are using the compile option USE_NETCDF ([here](https://github.com/DeniseWorthen/CICE/blob/0d8139d396d21b26614d7c103f888952c14a611c/configuration/scripts/forapps/ufs/comp_ice.backend.libcice#L62))

Reply:
Thanks for the explanation.